src/mnxmeth.c

```diff
@@ -1497,12 +1497,12 @@ MNXdrawpoint(LispArgs args) /* args[0] = LispPTR to MedleyWindow */
 }
 
 /**************************************************************/
-/* MNXdrawline                                                  */
+/* MNXdrawline                                                */
 /* color not yet implemented                                  */
 /* Medleys Drawline can't handle dashing.                     */
-/* Thick lines in Medley are handled as parallellograms       */
+/* Thick lines in Medley are handled as parallelograms        */
 /* instead of rectangles.                                     */
-/* Remember that this method has an OPERATION arument in Lisp */
+/* Remember that this method has an OPERATION argument in Lisp */
```

Comment:
I had line-wrapped `Lisp` down to the next line here too.

Reply:
Also fixed.